### ISSUE TITLE
Content picker doesn't close when clicking outside dialogs

### ISSUE THREAD

**Issue by ronaldbarendse** [KevinJump/Our.Umbraco.LinkedPages#1]
Just a minor thing: when opening the 'Linked Pages' dialog and adding a page (<kbd>Add +</kbd>), clicking outside the dialogs first closes the Linked Pages dialog, leaving the content picker open. Clicking a second time closes the content picker.

I would expect clicking outside the dialogs would first close the content picker or both dialogs at the same time. I don't know what the default UI experience should be in this case, I haven't seen similar dialogs in Umbraco...

**Comment by KevinJump** [comment]
yeah its a bit hacky, isn't it ? the behavior of the left 'context dialog' is very much governed by umbraco and when you click off it it closes, the other dialog is a bit more controllable. - there might be something we can do to hook into that one, and control what should happen.
